task_schedule.cfg

```diff
@@ -43,11 +43,11 @@ alert      aca@head.cfa.harvard.edu
       exec make_esaview_data.py --nweeks=6
```

Comment:
This would need the output changed per Tom's comment ("--data-root=$ENV{SKA_DATA}/acis_taco")